docs/build/wasp-evm/0.7.0/docs/getting-started/compatibility.md

```diff
@@ -0,0 +1,99 @@
+---
+description: Compatibility between the ISC EVM layer and existing Ethereum smart contracts and tooling.
+image: /img/logo/WASP_logo_dark.png
+keywords:
+  - smart contracts
+  - EVM
+  - Ethereum
+  - Solidity
+  - limitations
+  - compatibility
+  - fees
+  - reference
+---
+
+# EVM Compatibility in IOTA Smart Contracts
+
+The [`evm`](/wasp-wasm/reference/core-contracts/evm) [core contract](/wasp-wasm/reference/core-contracts/overview)
+provides EVM support in IOTA Smart Contracts. It stores the EVM state (account balances, state, code,
+etc.) and provides a way to execute EVM code to manipulate the state.
+
+The EVM core contract runs on top of the ISC layer, which provides the rest of the machinery needed to run smart
+contracts, such as signed requests, blocks, state, proofs, etc.
+
+However, the ISC EVM layer is also designed to be as compatible as possible with existing Ethereum tools
+like [MetaMask](https://metamask.io/), which assume that the EVM code runs on an Ethereum blockchain composed of
+Ethereum blocks containing Ethereum transactions. Since ISC works in a fundamentally different way,
+providing 100% compatibility is not possible. We do our best to emulate the behavior of an Ethereum node, so the
+Ethereum tools think they are interfacing with an actual Ethereum node, but some differences in behavior are inevitable.
+
+## Properties and Limitations
+
+:::warning
+There is a difference in the decimal precision of ether (18 decimal places) to MIOTA/SMR(6 decimal places). Because of this, when sending native tokens in the EVM, which are expressed in wei (ether = 10<sup>18</sup>wei), the last 12 decimal places will be ignored.
+
+example: 1,999,999,999,999,999,999 wei = 1.999,999 SMR/MIOTA
+:::
```

Comment:
```suggestion
:::warning

There is a difference in the decimal precision of ether (18 decimal places) to MIOTA/SMR(6 decimal places). Because of this, when sending native tokens in the EVM, which are expressed in wei (ether = 10<sup>18</sup>wei), the last 12 decimal places will be ignored.

Example: 1,999,999,999,999,999,999 wei = 1.999,999 SMR/MIOTA

:::
```